package.json

```diff
@@ -35,7 +35,7 @@
     "html5-history-api": "^4.2.7",
     "jasmine-core": "^2.2.0",
     "jquery": "^2.2.1",
-    "jquery-zoom": "^1.7.15",
+    "jquery-zoom": "BC-EChristensen/zoom#64651ce299804923b1f06184d368e9e22e79e4ec",
```

Comment:
We shouldn't point to a forked repo on a personal account. If we decide to fork this project (which I'm suspect of due to increased maintenance requirements), then it should be on the bigcommerce organization.